setup.py

```diff
@@ -16,8 +16,18 @@
 #
 # For further info, check https://github.com/canonical/charmcraft
 
+import ast
 import setuptools
 
+
+def get_version():
```

Comment:
why not `import charmcraft; return charmcraft.__version__`?

Reply:
Agreed.
finding a line in an __init__ script to literal_eval it seems odd here.